test/powershell/Modules/Microsoft.PowerShell.Utility/Select-String.Tests.ps1

```diff
@@ -71,6 +71,33 @@ Describe "Select-String" -Tags "CI" {
 	    $testinputone | Select-String -Pattern "goodbye" -NotMatch | Should -Be "hello", "hello"
 	}
 
+	it "Should return an array of matching strings with the first match highlighted when Emphasize is used" {
+		if ($IsWindows)
+		{
+			$testinputone | Select-String -Pattern "l" -Emphasize | Should -Be "he*l*lo", "he*l*lo"
```

Comment:
Imo emphasis is a formatting concern. The output of the cmdlet should be the raw data, and any highlighting should be done through PowerShell's formatting system (i.e. script blocks in `Format.ps1xml` files), so that piping to other cmdlets is still possible.

Examples:
https://github.com/felixfbecker/PowerGit#log
https://github.com/felixfbecker/PowerGit/tree/master/PowerGit/Formats